is/factory.ts

```diff
@@ -179,7 +179,18 @@ export function isTupleOf<
   }
 }
 
-type TupleOf<T> = {
+/**
+ * Tuple type of types that are predicated by an array of predicate functions.
+ *
+ * ```ts
+ * import { is, TupleOf } from "https://deno.land/x/unknownutil@$MODULE_VERSION/mod.ts";
+ *
+ * type A = TupleOf<readonly [typeof is.String, typeof is.Number]>;
+ * // Above is equivalent to the following type
+ * // type A = [string, number];
+ * ```
+ */
```

Comment:
I read the previous is.ts in passing and then copied and pasted it, so my comments may not be in line with the new implementation.